consensus/polybft/consensus_runtime.go

```diff
@@ -142,7 +132,7 @@ func (c *consensusRuntime) IsBridgeEnabled() bool {
 
 // AddLog is an implementation of eventSubscription interface,
 // and is called from the event tracker when an event is final on the rootchain
-func (c *consensusRuntime) AddLog(eventLog *ethgo.Log) {
+func (c *consensusRuntime) AddLog(eventLog *ethgo.Log) { //nolint
```

Comment:
Do we still need linting disable here? What was the problem here?

Reply:
Will be fixed in later on prs